src/calendar.ts

```diff
@@ -219,7 +219,10 @@ export const fetchCalendarObjects = async (params: {
     (
       await calendarQuery({
         url: calendar.url,
-        props: [{ name: 'getetag', namespace: DAVNamespace.DAV }],
+        props: [
+          { name: 'getetag', namespace: DAVNamespace.DAV },
+          { name: 'calendar-data', namespace: DAVNamespace.DAV },
```

Comment:
why include `calendar-data` at this step? 
this steps only fetches url of calendar objects
calendar data was intended to be fetched using `calendarMultiGet`

Reply:
You're right, it was a leftover from debugging something else